packages/expo-module-template/android/src/main/java/expo/modules/template/ModuleTemplateModule.kt

```diff
@@ -1,30 +1,19 @@
 package expo.modules.template
 
-import android.content.Context
+import expo.modules.kotlin.modules.Module
+import expo.modules.kotlin.modules.ModuleDefinition
 
-import org.unimodules.core.ExportedModule
-import org.unimodules.core.ModuleRegistry
-import org.unimodules.core.Promise
-import org.unimodules.core.interfaces.ExpoMethod
+class ModuleTemplateModule : Module() {
+  override fun definition() = ModuleDefinition {
+    name("ExpoModuleTemplate")
 
-class ModuleTemplateModule(context: Context) : ExportedModule(context) {
+    function("someGreatMethodAsync") { options: Map<String, String> ->
+      println("Hello 👋")
+      null as Any?
+    }
 
-  private var mModuleRegistry: ModuleRegistry? = null
-
-  override fun getName(): String {
-    return NAME
-  }
-
-  override fun onCreate(moduleRegistry: ModuleRegistry) {
-    mModuleRegistry = moduleRegistry
-  }
-
-  @ExpoMethod
-  fun someGreatMethodAsync(options: Map<String, Any>, promise: Promise) {
-  }
-
-  companion object {
-    private val NAME = "ExpoModuleTemplate"
-    private val TAG = ModuleTemplateModule::class.qualifiedName
+    viewManager {
+      view { ModuleTemplateView(it) }
```

Comment:
What would you say if we make it more clear what `it` means here? Can also use `applicationContext` or `activityContext` 🤷‍♂️ 
```suggestion
      view { context ->
        ModuleTemplateView(context)
      }
```